pyanaconda/kexec.py

```diff
@@ -73,6 +73,12 @@ def run_grubby(args=None):
         raise GrubbyInfoError("Missing values: %s" % ", ".join(boot_info_fields))
 
     boot_info = boot_info_class(**boot_info_args)
+
+    # There could be multiple initrd images defined for a boot entry, but
+    # the kexec command line tool only supports passing a single initrd.
+    if boot_info.initrd:
+        boot_info.initrd = boot_info.initrd.split(" ")[0]
```

Comment:
`boot_info` is a namedtuple, so I am afraid that this assignment will fail.

Reply:
@poncovka thanks, I missed that! I've pushed a new version that I think should work.